addons/website_slides_survey/static/src/xml/website_slide_upload.xml

```diff
@@ -1,29 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This template has not been replaced by 'website_slides_survey.SlideCategoryTutorial.Certification', which is simply the tutorial part of the 'certification' slide submission form.
This template is the old form.
In your commit you could remove the reference to 'website_slides_survey.SlideCategoryTutorial.Certification' and indicate that you're removing a template that we forgot to delete when we replaced the old slide_upload_dialog with a set of OWL components.